util/src/main/scala/scala/scalanative/io/ByteBufferPool.scala

```diff
@@ -7,7 +7,7 @@ final class ByteBufferPool {
   private var buffers: List[ByteBuffer] = Nil
 
   private def alloc(): ByteBuffer = {
-    ByteBuffer.allocateDirect(64 * 1024 * 1024)
+    ByteBuffer.allocateDirect(8 * 1024 * 1024)
```

Comment:
That one probably deserves its own ticket, feel free to file a new one with possible ideas for the solution. 👍